wlauto/workloads/blogbench/__init__.py

```diff
@@ -0,0 +1,78 @@
+#    Copyright 2015 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=attribute-defined-outside-init
+import os
+
+from wlauto import Workload, Parameter, Executable
+
+
+class BlogbenchWorkload(Workload):
+
+    name = 'blogbench'
+    description = """
+    Blogbench is a portable filesystem benchmark that tries to reproduce the
+    load of a real-world busy file server.
+
+    Blogbench stresses the filesystem with multiple threads performing random
+    reads, writes and rewrites in order to get a realistic idea of the
+    scalability and the concurrency a system can handle.
+
+    Source code are available from:
+        https://download.pureftpd.org/pub/blogbench/
+    """
+
+    parameters = [
+        Parameter('iterations', kind=int, default=30,
+                  description='The number of iterations to run'),
+        Parameter('directory', kind=str,
+                  default='/mnt/blogbench',
```

Comment:
Agree, updated the code, not sure if it is the right approach ;)
Please review.